packages/nvidia_nat_core/src/nat/front_ends/fastapi/fastapi_front_end_plugin_worker.py

```
@@ -1263,147 +1263,165 @@ class MCPToolInfo(BaseModel):
-                            "function_group": group_name,
-                            "server": client.server_name,
-                            "transport": config.server.transport,
-                            "session_healthy": session_healthy,
-                            "protected": True if config.server.auth_provider is not None else False,
-                            "tools": tools_info,
-                            "total_tools": len(configured_short_names),
-                            "available_tools": available_count
-                        })
+                                    override_configs[orig_name] = override
+                    except Exception:
+                        pass
```

Ruff (0.14.14)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)